client/webserver/site/src/css/market.scss

```diff
@@ -57,31 +59,35 @@ div.popupforms {
 }
 
 div.market-orders-table-container {
-  @media (min-width: 992px) {
+  $lg: map.get($grid-breakpoints, "lg");
+
+  @media (max-width: calc(#{$lg} - 1px)) {
```

Comment:
Bummer the mixing wasn't quite right.  I'm still uncertain why it wouldn't fit the bill here though.  Basically the "docs" at https://mdbootstrap.com/docs/standard/layout/breakpoints/ and the code in node_modules/bootstrap/scss/mixins/_breakpoints.scss indicate it's doing basically what you're doing except with a `- 0.02` instead of `- 1` for the `media-breakpoint-down` impl, which does still sound appropriate.

Reply:
The `$grid-breakpoints` variables that we are using in `application.scss` is different from the one in bootstrap. The `xxl` size we have is 1750px instead of 1400px. The `lg` size is the same but I think it's better to be consistent.

Reply:
Ok, I see your concern now.  But the map we define in application.scss seems to override the one in bootstrap's _variables.scss.  This seems to be on account of the use of the SASS `!default` flag (https://sass-lang.com/documentation/variables#default-values)

With the mixin:

![image](https://user-images.githubusercontent.com/9373513/130642677-f92520da-21a5-4e77-b26f-2b662054f815.png)

![image](https://user-images.githubusercontent.com/9373513/130642802-35b249c1-54d9-4155-8579-e671d9ae7f6a.png)

Reply:
Er, bootstrap link: https://getbootstrap.com/docs/4.3/getting-started/theming/#variable-defaults

Reply:
The issue was that I was using the same size in both the up and down case, not that it was using the incorrect variable. It's fixed now.

Reply:
Just making a note that when we update to Bootstrap 5, there's a breaking change we might not catch:

> Media query mixins parameters have changed for a more logical approach.
> 
>   -  media-breakpoint-down() uses the breakpoint itself instead of the next breakpoint (e.g., media-breakpoint-down(lg) instead of media-breakpoint-down(md) targets viewports smaller than lg).
>   - Similarly, the second parameter in media-breakpoint-between() also uses the breakpoint itself instead of the next breakpoint (e.g., media-between(sm, lg) instead of media-breakpoint-between(sm, md) targets viewports between sm and lg).
>